biosimulations/apps/combine-service/src/utils.py

```diff
@@ -49,7 +49,11 @@ def save_file_to_s3_bucket(filename):
         :obj:`str`: URL for saved file
     """
     # TODO
-    return 'https://data.biosimulations.org/XYZ'
+    temp_dirname = os.path.join('/tmp', 'combine-service-static')
+    if not os.path.isdir(temp_dirname):
```

Comment:
@jonrkarr Could you take a look at the security warning and confirm that the usage is fine?

Reply:
https://security.openstack.org/guidelines/dg_using-temporary-files-securely.html

Reply:
https://docs.openstack.org/bandit/1.4.0/plugins/hardcoded_tmp_directory.html

Reply:
This temporary just so I could test its working. This is intended to be replaced by your #2411.

Reply:
Whoops, completely missed that.